docs/notebook-demos/layers.ipynb

```diff
@@ -659,13 +712,15 @@
    "source": [
     "### Reproject\n",
     "\n",
-    "This is similar to the `reproject` method for `RasterLayer` where the reprojection will not sample past the tiles' boundaries. Because of this, whatever `zoom_level` the `TiledRasterLayer` has will be changed to 0 since the area being represented changes to just the tiles."
+    "This is similar to the `reproject` method for `RasterLayer` where the reprojection will not sample past the tiles' boundaries. This means the layout of the tiles will be changed so that they will take on a `LocalLayout` rather than a `GlobalLayout`. Because of this, whatever `zoom_level` the `TiledRasterLayer` has will be changed to 0 since the area being represented changes to just the tiles."
```

Comment:
Ah, forgot to point that in. No, I've never added a section on the importance of `zoom` level. I only added that part as a way to let people know that the zoom of their layer will change so they wouldn't be surprised.